### PR TITLE
Implemented news cards and enchants

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: -

* **Server Mode**: renewal

* **Description of Pull Request**: 

Implemented news cards and enchants according to KRO description.

### CODE REVIEW

**Comment by idk-whoami** [comment]
some item additional bonus 2% for enchantgrade B or A
E.g. IDs 310675, 310676

**Comment by Atemo** [comment]
Yeah and I didn't know if the previous enchantment level should work (they work), for example if the grade is B the bonus in grade D should be triggered too

**Comment by attackjom** [comment]
> Yeah and I didn't know if the previous enchantment level should work (they work), for example if the grade is B the bonus in grade C.. should be triggered too

Yes. It increase by addition.
if you have gradeA you will get all the bonus in previous.

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -83146,6 +83100,52 @@ Body:
...
+         }
+         if (readparam(bInt) >= 70) {
+            bonus bMatk,5;
+            if (.@r >= 7)
+               bonus bMatk,10;
+         }
+         if (readparam(bVit) >= 70) {
+            bonus bMaxHPrate,2;
+            bonus bMaxSPrate,2;
+            if (.@r >= 7)
```
Comment (Litro) [inline db/re/item_db_equip.yml:83130]: is this intended or missing bracket?

Comment (Atemo) [reply db/re/item_db_equip.yml:83130]: You're right the bracket seems missing !
https://github.com/rathena/rathena/pulls

**Pull request merged**